Loop/Models/RemoteCommand.swift

```diff
@@ -8,78 +8,34 @@
 
 import Foundation
 import LoopKit
-import HealthKit
-
-public enum RemoteCommandError: LocalizedError {
-    case expired
-    case invalidOTP
-    case missingMaxBolus
-    case exceedsMaxBolus
-    case exceedsMaxCarbs
-    case invalidCarbs
-    
-    public var errorDescription: String? {
-        get {
-            switch  self {
-            case .expired:
-                return NSLocalizedString("Expired", comment: "Remote command error description: expired.")
-            case .invalidOTP:
-                return NSLocalizedString("Invalid OTP", comment: "Remote command error description: invalid OTP.")
-            case .missingMaxBolus:
-                return NSLocalizedString("Missing maximum allowed bolus in settings", comment: "Remote command error description: missing maximum bolus in settings.")
-            case .exceedsMaxBolus:
-                return NSLocalizedString("Exceeds maximum allowed bolus in settings", comment: "Remote command error description: bolus exceeds maximum bolus in settings.")
-            case .exceedsMaxCarbs:
-                return NSLocalizedString("Exceeds maximum allowed carbs", comment: "Remote command error description: carbs exceed maximum amount.")
-            case .invalidCarbs:
-                return NSLocalizedString("Invalid carb amount", comment: "Remote command error description: invalid carb amount.")
-            }
-        }
-    }
-}
-
-
-enum RemoteCommand {
-    case temporaryScheduleOverride(TemporaryScheduleOverride)
-    case cancelTemporaryOverride
-    case bolusEntry(Double)
-    case carbsEntry(NewCarbEntry)
-}
-
 
 // Push Notifications
```

Comment:
Generally like before, this method parses a notification and extracts a remote action (previously command).

The difference is Loop valid action checks are not done here (ex: Carb amount within reasonable limits). That will instead be managed by the client. 

Separating notification payload parsing and validation seems like a good idea here. This will go away when I use Codable to do all this parsing in Remote 2.0.